src/test/java/com/geronimo/GeronimoDatabaseTest.java

```diff
@@ -0,0 +1,26 @@
+package com.geronimo;
+
+import com.geronimo.dao.UserRepository;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class GeronimoDatabaseTest {
+
+    private UserRepository userRepository;
+
+    @Test
+    public void testDatabase() {
+
```

Comment:
the test to check that database is working could be just executing a query "SELECT 1" on that database using an entityManager's native SQL query support. I believe there's something for native query execution.

Reply:
I used autowired userRepository to check if most things are working as supposed to, but later deleted all code from "testDatabase" method. Though, thanks for that piece of advice